docs/misc/asciinema/Vagrantfile

```diff
@@ -15,7 +15,7 @@ Vagrant.configure("2") do |config|
   SHELL
   config.vm.provision "record basic usage", type: "shell", inline: <<-SHELL
     # `rm` below allows quick re-exec via:
-    # vagrant vagrant provision --provision-with "record basic usage"
+    # vagrant provision --provision-with "record basic usage"
```

Comment:
I'm _pretty_ sure this doubling is an error.

Reply:
yeah